bots/task/github.py

```diff
@@ -325,6 +300,26 @@ def commits(self, branch='master', since=None):
                 yield commit
                 count += 1
 
+    def whitelist(self):
+        users = set()
+        teamId = self.teamIdFromName(TEAM_CONTRIBUTORS)
+        page = 1
+        count = 100
+        while count == 100:
+            data = self.get("/teams/{0}/members?page={1}&per_page={2}".format(teamId, page, count)) or []
+            users.update(user.get("login") for user in data)
+            count = len(data)
+            page += 1
+        return users
+
+    def teamIdFromName(self, name):
+        for team in self.get("/orgs/cockpit-project/teams") or []:
+            if team.get("name") == TEAM_CONTRIBUTORS:
```

Comment:
This should be name, the parameter, I think, and not TEAM_CONTRIBUTORS.

Reply:
Woops, thanks for catching that. My fault for just copying.